src/storage/src/manifest/storage.rs

```diff
@@ -94,30 +115,77 @@ impl LogIterator for ObjectStoreLogIterator {
 #[derive(Clone, Debug)]
 pub struct ManifestObjectStore {
     object_store: ObjectStore,
+    compress_type: CompressionType,
     path: String,
 }
 
 impl ManifestObjectStore {
     pub fn new(path: &str, object_store: ObjectStore) -> Self {
         Self {
             object_store,
+            compress_type: CompressionType::ZSTD,
             path: util::normalize_dir(path),
         }
     }
 
     #[inline]
+    /// return delta file path under **current** compress algorithm
```

Comment:
nit: Let's use the declarative sentence
```suggestion
    /// Returns delta file path under **current** compress algorithm
```